pkg/security/probe/probe_linux.go

```diff
@@ -436,8 +436,8 @@ func (p *Probe) SendStats() error {
 	for evtType, count := range p.anomalyDetectionSent {
```

Comment:
Should the `anomalyDetectionSent` variable be renamed to match the metric's name?

Reply:
Sure !